js/admin.js

```diff
@@ -4,11 +4,18 @@
 
 jQuery( document ).ready(
 	function( $ ) {
-		var transitionTimeout;
+		// Add a boolean variable to be able to check jQuery UI >= 1.12 which is introduced in WP 5.6.
+		// Backward compatibility WP < 5.6
+		var isJqueryUImin112 = $.ui.version >= '1.12.0';
+		// Allow to check if a flag list dropdown is present. Not present in the Wizard steps or other settings page.
+		var flagListExist = $( "#flag_list" ).length;
+		// Allow to check if a language list dropdown is present. Not present in other settings page.
+		var langListExist = $( "#lang_list" ).length;
 
```

Comment:
For the languages list dropdown, maybe.
However it isn't enough because we also need the languages list dropdown in some Wizard steps where there is no flags list dropdown.